src/editor/components/EditorElement.tsx

```diff
@@ -49,36 +57,47 @@ export const EditorElement: React.FC<EditorElementProps> = ({
   children,
 }) => {
   const schema = useSchema();
-  const elementSchemaPathString = wrappedElement.linkedSchemaElements?.find(
-    (schemaElement) => schemaElement !== undefined
-  );
-  const elementSchemaPath = elementSchemaPathString?.split('/');
-  const elementSchema = elementSchemaPath
-    ? getFromPath(schema, elementSchemaPath)
-    : undefined;
-
+  const [selection, setSelection] = useSelection();
+  const dispatch = useDispatch();
   const [openConfirmRemoveDialog, setOpenConfirmRemoveDialog] = React.useState(
     false
   );
+  const schemaElementUUID = wrappedElement.linkedSchemaElements?.values().next()
+    .value;
+  const searchResult = schemaElementUUID
+    ? findByUUID(schema, schemaElementUUID)
+    : undefined;
+  const elementSchema = isUUIDError(searchResult) ? undefined : searchResult;
```

Comment:
Partially related to this PR: As we hardcode `linkedSchemaElements` to be of only size one, we should probably refactor it to be a simple property directly containing only one uuid. Also we can avoid resolving here but just let the reducer handle it, the reducer itself can take the uuid out of the uiSchemaElement and then do whatever it wants with it.